android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -84,6 +84,18 @@ public void setAdFormat(final AppLovinMAXAdView view, final String adFormatStr)
         view.setAdFormat( adFormatStr );
     }
 
+    @ReactProp(name = "extraParameters")
+    public void setExtraParameters(final AppLovinMAXAdView view, @Nullable final ReadableMap value)
+    {
+        view.setExtraParameters( value );
+    }
+
+    @ReactProp(name = "localExtraParameters")
```

Comment:
Let's walk through the PR and correct the plurality of our APIs. Since we're setting one/singular parameter, it should be `setLocalExtraParameter()` and `setExtraParameter()`, without the `s`.

Reply:
Let me make sure that both APIs for functions and UIs should be singular:

Functions:
- setExtraParameter
- setLocalExtraParameter

UI components:
- @ReactProp(name = "extraParameter")
- @ReactProp(name = "localExtraParameter")